### PR TITLE
ada-url@2.9.2

### PR DESCRIPTION
**Pull request by c8ef**

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (ada) have been updated in this PR. Please review the changes.

File: modules/ada/2.9.2/MODULE.bazel [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module(
+    name = "ada",
+    version = "2.9.2",
+    compatibility_level = 1,
```
Comment (Wyverald) [inline modules/ada/2.9.2/MODULE.bazel:4]: please add `bazel_compatibility = [">=7.2.1"],` here, for the usage of overlay files

**Review by Wyverald — APPROVED** [review]
could we consider calling this module `ada_url` or `ada-url`, maybe? I think the first `ada` that pops into people's minds would probably be the programming language.

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (ada-url) have been updated in this PR. Please review the changes.

**Review by meteorcloudy — APPROVED** [review]

**Comment by anonrig** [comment]
Is there any plans to implement auto-updaters to the registry? @meteorcloudy @Wyverald

**Comment by meteorcloudy** [comment]
I guess you mean auto-updating a C++ Bazel module when there is a new release? If the original project author wants, they can integrate https://github.com/bazel-contrib/publish-to-bcr. If not, then the community module maintainer has to do this manually, but can be helped with https://github.com/bazelbuild/bazel-central-registry/blob/main/GEMINI.md

**Pull request merged**